src/parser.rs

```diff
@@ -1474,6 +1529,25 @@ fn collect_definitions(seg: &mut Segment) {
                     name: copy_token(math[1].as_ref(buf)),
                 });
             }
+            HeadingComment(level) => {
+                seg.outline.push(HeadingDef { 
+					name: copy_token(get_heading_name(buf,stmt.span.start)),
+					index, 
+					level 
+				});
+            }
+			AdditionalInfoComment => {
```

Comment:
There is a mix of tabs and spaces here.

Reply:
I'll make uniform use of 4 spaces until we run rust fmt.